thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/WebappConfigResource.java

```diff
@@ -0,0 +1,142 @@
+package com.linkedin.thirdeye.dashboard.resources;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.linkedin.thirdeye.dashboard.configs.AbstractConfig;
+import com.linkedin.thirdeye.dashboard.configs.WebappConfigClassFactory;
+import com.linkedin.thirdeye.dashboard.configs.WebappConfigClassFactory.WebappConfigType;
+import com.linkedin.thirdeye.db.dao.WebappConfigDAO;
+import com.linkedin.thirdeye.db.entity.WebappConfig;
+
+@Path(value = "/dashboard")
```

Comment:
put a unique name, there is already an end point with 'dashboard'. This may cause a conflict in end points. 
you may rename this to 'dashboard/config'